CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 ## next
 
+
+* Added `FieldExpander`, a new class that recursively expands a tree of
+  of attributes into their lowest-level values.
+    * For example, for the `Person` and `Address` blueprints defined in
+    [spec_blueprints.rb](sped/support/spec_blueprints.rb):
+      * `FieldExpander.expand(Person, {name: true, full_name: true})` would return
+        `{name: true, full_name: {first: true, last:true}}`.
+    * Attempting to expand attributes with circular references (i.e. the
+      a person's address, where that address then references that person) will
+      raise a `Praxis::FieldExpander::CircularExpansionError` error.
+* Added `Renderer`, a new class for rendering objects (typically Blueprint
```

Comment:
general thought. Should we call the `Renderer` class, `FieldRenderer` instead? ... reading this made me realize that it might be a little misleading.